docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -1,40 +1,44 @@
 [[synthetics-create-test]]
-= Create a new synthetic test
+= Write a synthetic test
 
 ++++
-<titleabbrev>Create a test</titleabbrev>
+<titleabbrev>Write a synthetic test</titleabbrev>
```

Comment:
I guess one confusing challenge here is that inline journeys only use the `step` syntax (plus playwright). That context feels important here to me, maybe we need a preamble explaining that? I think just seeing a couple of examples would be most clear. We do explain this more later, but it'd be good to have some sort of context here.

Your thoughts?

Reply:
IMHO that might be a bit too much information up-front. I do think we should explain that to users, but one thing I always like is when documentation only exposes me to what I need to know at that particular moment.

To summarise, when I was writing my book, my publisher's advice was "just in time, not just in case". In this case, I think we'd be including this information "just in case", not because it's necessary at this particular moment.

If I were to make the call, I'd add that information as close as possible to where we explain how to create a monitor with an inline journey in Heartbeat or the UI. I also think it would be worth using a callout so it's out of the main text flow.

Reply:
I think I can see this one both ways. I won't block the review on this, I'm +1 on leaving up to @colleenmcginnis 's judgement here